libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -132,6 +132,16 @@ def add_texts(
         # Insert the embedded texts in the vector store table.
         return self._insert_embeddings(texts, embedded_texts, metadatas, ids)
 
+    def drop(self) -> None:
```

Comment:
Do we need to call drop in a destructor?

Reply:
I had made a response to this in the previous PR. Adding it here, since this PR is mainly for the `drop` function.

If we use invoke this in the destructor, every time the `SQLServer_VectorStore` object is to be destroyed, the vector store table gets deleted. This isn't the behavior we want, since we want users to be able to make that decision on if the vector store table should be deleted or not, hence the reason why we only create a function that does that action.

However, now that it's mentioned we should clean up our session resources in the destructor.